images/wordpress/wp-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+if test ${WORDPRESS_VERSION}X = latestX; then
+  wp core download --path=/var/www/html
+else
```

Comment:
I was thinking that the `WORDPRESS_VERSION` variable would mainly be used for when we need to install an older version of WordPress for debugging (and then we'd remove it from the docker-compose.yml file without committing). I hadn't considered the case where somebody would use `WORDPRESS_VERSION=4.8.1` (where 4.8.1 is the current version).

We can probably catch that case quite easily, using something along these lines:

```
elif test ${WORDPRESS_VERSION}X = `wp core version --path=/var/www/html`X; then
  # do nothing
```